### PR TITLE
// Cleaning some Core/Foundation classes

### PR DESCRIPTION
**Pull request by xGouley**

### CODE REVIEW

File: classes/Customer.php [hunk 1]
```diff
@@ -329,7 +329,7 @@ public function getByEmail($email, $passwd = null, $ignore_guest = true)
             '.Shop::addSqlRestriction(Shop::SHARE_CUSTOMER).' AND `deleted` = 0 AND `is_guest` = 0');
 
         try {
-            $crypto = \PrestaShop\PrestaShop\Adapter\ServiceLocator::get('\\PrestaShop\\PrestaShop\\Core\\Foundation\\Crypto\\Hashing');
+            $crypto = \PrestaShop\PrestaShop\Adapter\ServiceLocator::get('\\PrestaShop\\PrestaShop\\Core\\Crypto\\Hashing');
```
Comment (Shudrum) [inline classes/Customer.php]: Why not add a use statement ?

```php
use PrestaShop\PrestaShop\Adapter\ServiceLocator;
```

Comment (xGouley) [inline classes/Customer.php]: Used to let this to emphasize it's an Adapter...

File: classes/Customer.php [hunk 2]
```diff
@@ -329,7 +331,7 @@ public function getByEmail($email, $passwd = null, $ignore_guest = true)
             '.Shop::addSqlRestriction(Shop::SHARE_CUSTOMER).' AND `deleted` = 0 AND `is_guest` = 0');
 
         try {
-            $crypto = \PrestaShop\PrestaShop\Adapter\ServiceLocator::get('\\PrestaShop\\PrestaShop\\Core\\Foundation\\Crypto\\Hashing');
+            $crypto = ServiceLocator::get('\\PrestaShop\\PrestaShop\\Core\\Crypto\\Hashing');
         } catch (\PrestaShop\PrestaShop\Adapter\CoreException $e) {
```
Comment (Shudrum) [inline classes/Customer.php]: Hey! Can I abuse?

Maybe you can also call `use` here?

Comment (xGouley) [inline classes/Customer.php]: abuses

**Comment by Shudrum** [comment]
:+1:

**Pull request merged**